docs/api/drivers/ResetReason.md

```diff
@@ -0,0 +1,43 @@
+## ResetReason
+
+When the system restarts, the system registers contain the reason for the restart at boot time in a platform specific manner. This API provides a generic method of fetching the reason for the restart.
+
+You can use the ResetReason interface to determine the cause of the last system reset in a portable fashion.
+
+### ResetReason class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/mbed-os/development/mbed-os-api-doxy/_reset_reason_8h_source.html)
```

Comment:
I've found the .h file in the Doxygen, but I can't find the class reference. I thought it would be here: https://os.mbed.com/docs/mbed-os/development/mbed-os-api-doxy/annotated.html but I don't see it.

Reply:
I'm not sure if I can help here. Taking a `Ticker` class as an example, the `v5.13` doxy can be found here:
`https://os.mbed.com/docs/mbed-os/v5.13/mbed-os-api-doxy/classmbed_1_1_ticker.html`

So I think we should expect `ResetReason` in `NEW_DOCS_VERSION`:
`https://os.mbed.com/docs/mbed-os/NEW_DOCS_VERSION/mbed-os-api-doxy/classmbed_1_1_reset_reason.html`
and `Watchdog` in:
`https://os.mbed.com/docs/mbed-os/NEW_DOCS_VERSION/mbed-os-api-doxy/classmbed_1_1_watchdog.html`

At least these are the filenames I get when generating the doxygen locally.

Reply:
The one for Watchdog is here: https://os.mbed.com/docs/mbed-os/development/mbed-os-api-doxy/classmbed_1_1_watchdog.html, but I can't find the one for ResetReason. There may be something wrong with the doxyfile_options files. @0xc0170 Can you help?

Reply:
I think https://github.com/ARMmbed/mbed-os/pull/10990 should fix this.

Reply:
That worked. 👍 Thanks for the PR.